### PR TITLE
New models and Controllers

### PR DESCRIPTION
**Pull request by fedebotu**
Add new models:
1. Classic control: spring-mass, linear time-invariant system, inverted pendulum, acrobot, cart-pole 
2. Continuous Stirred Tank Reactor (CSTR)
3. Quadcopter

Add new controllers:
- Neural controllers
- MPC

Others:
- Tutorials on direct and model predictive control
- Utilities
- Cost function
- Plotting
- Miscellaneuous

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+torchdyn
```
Comment (Zymrael) [inline requirements.txt:1]: We should add minimum versions requirements.

**Comment by Zymrael** [comment]
LGTM, make sure to add more information on the requirements in the next PR. Are we planning to release this on `PyPI` for `pip` install?

**Comment by Zymrael** [comment]
Also: now that MuJoCo is open-source, let's produce a few examples using our controller API!

**Comment by fedebotu** [comment]
@Zymrael sure, next PR I shall put more info about the requirements. With some more material, a PyPI release would be great! 🚀
As for MuJoCo, added on the list of todos ✅. 
Would be also great if we found extra contributors :)

**Pull request merged**